tasks/sync-granule/tests/sync_granule_test.js

```diff
@@ -175,7 +175,7 @@ test.serial('no error when collection info is not provided in the event', async
   t.is(output.granules[0].files.length, 1);
 });
 
-test.serial('download Granule from FTP endpoint', async (t) => {
+test.serial.only('download Granule from FTP endpoint', async (t) => {
```

Comment:
Remove `.only`

Reply:
🤦   I'm going to put a hook in git for this.   Thanks. 

857ce9d6f